packages/test-helpers/package.json

```diff
@@ -1,20 +1,24 @@
 {
-  "name": "@aragon/contract-helpers-test",
-  "version": "0.0.2",
+  "name": "@aragon/test-helpers",
```

Comment:
Keep using the same `test-helpers` namespace. Probably we will do a major release?

Reply:
I've been thinking that `test-helpers` is a bit too generic, since we have frontend and contract testing.

Maybe we can publish under `@aragon/contract-test-helpers`? And in the future we can have `@aragon/fe-test-helpers` (or similar)?